lib/tasks/download-assets.js

```diff
@@ -55,7 +55,13 @@ export default function downloadAssets (options) {
       }
       const locales = Object.keys(asset.fields.file)
       return Promise.mapSeries(locales, (locale) => {
-        const url = asset.fields.file[locale].url || asset.fields.file[locale].upload
+        const url = asset.fields.file[locale].url
+        if (!url) {
+          task.output = `${figures.warning} asset '${getEntityName(asset)}' doesn't contain an url in path asset.fields.file[${locale}].url`
+          warningCount++
```

Comment:
I would consider this an error, not a warning. (If they are trying to download all their assets and one of them isnt downloadable, that's an error they should be aware of.)